h/emails/reply_notification.py

```diff
@@ -33,10 +31,9 @@ def generate(request, notification):
         'document_title': document_title,
         'document_url': notification.parent.target_uri,
         'parent': notification.parent,
-        'parent_url': parent_url,
         'parent_user': notification.parent_user,
         'reply': notification.reply,
-        'reply_url': reply_url,
+        'reply_url': links.incontext_link(request, notification.parent),
```

Comment:
It happens to work either way currently (since `incontext_link()` returns a direct link to the thread root anno, any anno in the thread will do as input and give the same output) but I think passing in `notification.reply` is semantically correct so I've changed it to that